File: wireless/devices/instance-get-example-1/instance-get-example-1.py
# Download the **Next-Gen** twilio-python library from:
# twilio.com/docs/libraries/python#installation-nextgen
from twilio.rest import Client

account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" # Your Account SID from www.twilio.com/console
auth_token  = "your_auth_token"  # Your Auth Token from www.twilio.com/console

client = Client(account_sid, auth_token)
wireless = client.preview.wireless

d = wireless.devices('DEd3f4ec6c85ded1a5f64f10afc067895c').fetch()
print(d.friendly_name)

File: wireless/commands/instance-get-example-1/instance-get-example-1.php
<?php
// Download the **Next-Gen** twilio-php library from:
// twilio.com/docs/libraries/php#installation-nextgen

// Include dependencies from Composer's autoload (including Twilio library)
include 'vendor/autoload.php';

$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";  // Your Account SID from www.twilio.com/console
$token = "your_auth_token"; // Your Auth Token from www.twilio.com/console

$client = new Twilio\Rest\Client($sid, $token);

$cmd = $client->preview->wireless->commands("DCcb6aa7c620874dd98ba6c9c4108d0ed0")->fetch();

echo $cmd->command, "\n";